browser/net/brave_system_request_handler_unittest.cc

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/net/brave_system_request_handler.h"
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "brave/browser/net/url_context.h"
+#include "brave/common/network_constants.h"
+#include "chrome/test/base/chrome_render_view_host_test_harness.h"
+#include "net/traffic_annotation/network_traffic_annotation_test_helper.h"
+#include "net/url_request/url_request_test_util.h"
+#include "services/network/public/cpp/resource_request.h"
+
+#if !defined(BRAVE_SERVICES_KEY)
+#define BRAVE_SERVICES_KEY "dummytoken"
+#endif
+
+namespace brave {
+
+class BraveSystemRequestHandlerTest : public testing::Test {
```

Comment:
you don't need to have an empty class, just use `TEST(BraveSystemRequestHandlerTest, AddBraveServiceKeyHeader)` for tests